source/dub/commandline.d

```diff
@@ -997,7 +1048,13 @@ class InitCommand : Command {
 			}
 			p.description = input("Description", p.description);
 			p.authors = input("Author name", author).split(",").map!(a => a.strip).array;
-			p.license = input("License", p.license);
+			p.license = select(
+				"a license (https://choosealicense.com/)",
+				free_choice,
+				p.license,
+				"MIT", "BSL-1.0 (Boost)", "public domain", "GPL-3.0", "BSD 3-clause", "BSD 2-clause",
+				"GPL-2.0", "LGPL-3.0", "Apache-2.0", "MPL-2.0 (Mozilla)", "AGPL-3.0", "proprietary"
```

Comment:
Artistic-v2 could be added (I use it).

Also, the CC's although there should be a warning asking are you sure?

Reply:
I think I used the license list from the PR I rebased here, it probably originates from https://dub.pm/package-format-json.html#licenses, although there are a few more there.

Reply:
I have sorted by usage popularity now, didn't actually see Artistic-v2 used in any published DUB projects yet, do you have one?

You can also still manually type in text, so you can manually type Artistic-v2